msteams-platform/concepts/build-and-test/manage-your-apps-in-developer-portal.md

```diff
@@ -0,0 +1,158 @@
+---
+title: Manage your apps with the Developer Portal
+description: In this article, learn how to configure, distribute, and manage your apps using the Developer Portal for Microsoft Teams.
+ms.localizationpriority: medium
+ms.topic: overview
+ms.author: surbhigupta
+---
+
+# Manage your apps in Developer Portal
+
+After you create or upload your app, you can manage your apps in Developer Portal with the following:
+
+* [Overview](#overview)
+* [Configure](#configure)
+* [Advanced](#advanced)
+* [Publish](#publish)
+
+## Overview
+
+In the **Overview** section, you can see the following components to manage your app:
+
+* Dashboard
+
+  * In the **Dashboard** under **Overview** section, you can see the following components for your app:
+    * **Teams store validation**: App validation tool checks your app package against the test cases Microsoft uses when reviewing your app.
+    * **Announcement**: Latest updates of your apps on Developer Portal for Teams
+    * **Active users (Preview)**: Shows you the active user count
+    * **Basic information**: Shows you the App ID, Version, Manifest version, and so on.
+
+    :::image type="content" source="../../assets/images/tdp/dashboard-page.PNG" alt-text="The screenshot is an example that shows the Overview page of the app you created in Developer Portal for Teams.":::
+
+* Analytics
+
+    In the **Analytics** page under **Overview** section, you can see the total number of active users for your app. For more information, see [Analyze your app's usage](analyze-your-apps-usage-in-developer-portal.md).
+
+## Configure
```

Comment:
Configure what?

Reply:
UI component under Apps page to manage apps in developer portal, and it is explained in the start.